src/deavinaSJun20/FibbonacciSeries.java

```diff
@@ -0,0 +1,27 @@
+package deavinaSJun20;
+
+import java.util.Scanner;
+
+public class FibbonacciSeries {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please enter input to display first n Fibonaaci Numbers: ");
+		int num = scanner.nextInt();
+		new FibbonacciSeries().fibbonacciSeries(num);
+		scanner.close();
+	}
+
+	private void fibbonacciSeries(int num) {
+		int firstnum = 0;
```

Comment:
Fibbonacci series starts from 1. So please change your firstnum to 1

Reply:
we can start it from 0 also.
I have updated the code as per the review comments. Thanks

Reply:
Now the code looks good to me.